CONTRIBUTING.md

```diff
@@ -1,35 +1,44 @@
 # Contributing
 
-Everyone is welcome to contribute to containers. Reach out to team members if you have questions:
+## Requirements
 
-- Matrix chat: [#containers:mozilla.org](https://matrix.to/#/#containers:mozilla.org)
-- Email: containers@mozilla.com
+* Firefox 91.1.0+
+* Git 2.13+
+* Node 7+
 
-## Filing bugs
+## Getting Started
 
-If you find a bug with containers, please file a issue.
+1. Follow the instructions on [How to fork a repository][fork]
+2. Fetch the locales:
 
-Check first if the bug might already exist: https://github.com/mozilla/multi-account-containers/issues
+    ```
+    cd multi-account-containers
+    git submobule update --init
+    ```
+3. Install the project dependencies
+    ```
+    npm install -legacy-peer-deps
+    ```	
+3. Install [web-ext][web-ext] command-line tool:
 
-[Open an issue](https://github.com/mozilla/multi-account-containers/issues/new)
+    ```
+    npm install -g web-ext
```

Comment:
Suggestion: Let's remove making install line. We can just have users install deps and run the [`npm run web-ext`](https://github.com/mozilla/multi-account-containers/blob/main/package.json#L39) command from the `package.json`

Reply:
So you'd prefer the user to use the local `web-ext` bin than the global one right?

Reply:
Agreed!